plsync/mlabconfig.py

```diff
@@ -111,8 +269,11 @@ def main():
     elif options.format == 'sitestats':
         export_mlab_site_stats(sys.stdout, sites)
     elif options.format == 'zone':
-        print 'Sorry, mlabconfig does not yet support generating zone files.'
-        sys.exit(1)
+        with open(options.zoneheader, 'r') as header:
```

Comment:
"The circle is now complete." -- https://www.youtube.com/watch?v=EZD1f9RTEJU 

It means I should check whether the file exists before now. So, I've added a check to `parse_flags`. :-)